core/src/main/scala/kafka/server/ReplicaFetcherThread.scala

```diff
@@ -211,10 +218,20 @@ class ReplicaFetcherThread(name: String,
   }
 
   protected def fetch(fetchRequest: FetchRequest): Seq[(TopicPartition, PartitionData)] = {
-    val clientResponse = leaderEndpoint.sendRequest(fetchRequest.underlying)
-    val fetchResponse = clientResponse.responseBody.asInstanceOf[FetchResponse]
-    fetchResponse.responseData.asScala.toSeq.map { case (key, value) =>
-      key -> new PartitionData(value)
+    debug("ReplicaFetcherThread#fetch({})" + fetchRequest.underlying)
+    try {
+      val clientResponse = leaderEndpoint.sendRequest(fetchRequest.underlying)
+      val fetchResponse = clientResponse.responseBody.asInstanceOf[FetchResponse]
+      debug("ReplicaFetcherThread#fetch(" + fetchRequest.underlying +
```

Comment:
nit: use string substitutions.

Reply:
Oops.  This log message should have been removed.  It duplicates some TRACE log messages in the network handling code that print out the request and response.  I'll get rid of it now